draft-thomson-webpush-protocol.xml

```diff
@@ -637,7 +637,11 @@ HEADERS      [stream 4] +END_STREAM
 ]]>
           </artwork>
         </figure>
-
+        <t>
+          The push server MUST generate a 5XX (TBD) if the user agent fails to
```

Comment:
I think that this needs to be "MUST push a response with a status code of 5XX" instead.